### PR TITLE
Consistently use User.get_default_admin

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: src/api/spec/jobs/consistency_check_job_spec.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@
     let!(:project) { create(:project, name: 'super_project') }
     let(:consistency_checkjob) { described_class.new }
     let(:error_message) { "Project meta is different in backend for super_project\n{:foo=>\"bar\"}" }
+    let!(:admin_user) { create(:admin_user, login: 'Admin') }
```
Comment (krauselukas) [inline src/api/spec/jobs/consistency_check_job_spec.rb:31]: @hennevogel why do you had to add this now? The logic of the code is stayed the same, just curious

File: src/api/spec/support/database_cleaner.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@
     # For feature test we use deletion instead of transactions because the
     # test suite and the capybara driver do not use the same server thread.
     DatabaseCleaner.strategy = if example.metadata[:type] == :feature || example.metadata[:type] == :migration || example.metadata[:thinking_sphinx] == true
-                                 # Omit truncating what we have set up in db/seeds.rb except users and roles_user
```
Comment (dmarcoux) [inline src/api/spec/support/database_cleaner.rb:21]: This comment wasn't true or it changed? I don't see how this changed.

File: src/api/spec/jobs/consistency_check_job_spec.rb (hunk 1 above)
Comment (hennevogel) [reply src/api/spec/jobs/consistency_check_job_spec.rb:31]: We truncate the users table between test runs, including the Admin user.

File: src/api/spec/support/database_cleaner.rb (hunk 2 above)
Comment (hennevogel) [reply src/api/spec/support/database_cleaner.rb:21]: Yeah there are way more things we don't truncate. And we *do* truncate `users`.

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**